### PR TITLE
feat: add submitted packages

### LINKED ISSUE #425: 【パッケージの更新】レベルメーター
### 名前
sakana

### データ
``` xml
[
  {
    "id": "hebiiro/LevelMeter",
    "developer": "hebiiro",
    "downloadURLs": [
      "https://github.com/hebiiro/AviUtl-Plugin-LevelMeter/releases/latest"
    ],
    "name": "レベルメーター",
    "pageURL": "https://github.com/hebiiro/AviUtl-Plugin-LevelMeter/releases/latest",
    "overview": "音声のレベルメーター",
    "description": "Aviutl内に音声のレベルメーターを表示します。",
    "latestVersion": "1.3.0"
  }
]
```

### タイムスタンプ
2022/12/06 1:31:37

### LINKED ISSUE #435: ちょっとノリノリスクリプト
## パッケージ名
ちょっとノリノリスクリプト

## 概要
https://www.nicovideo.jp/watch/sm33151219

## 説明
https://www.nicovideo.jp/watch/sm33151219

## 開発者
おばけ大根

## 紹介ページURL
https://www.nicovideo.jp/watch/sm33151219

## ダウンロードページURL
https://www.nicovideo.jp/watch/sm33151219

### LINKED ISSUE #436: 【パッケージの更新】カラーコード追加
### 名前
匿名だれか

### データ
``` xml
[
  {
    "id": "hebiiro/AddColorCode",
    "developer": "hebiiro",
    "name": "カラーコード追加",
    "downloadURLs": [
      "https://github.com/hebiiro/AviUtl-Plugin-AddColorCode/releases/latest"
    ],
    "pageURL": "https://github.com/hebiiro/AviUtl-Plugin-AddColorCode#readme",
    "overview": "色の選択ダイアログでカラーコードを使えるようにします。",
    "description": "色の選択ダイアログでカラーコードを使えるようにします。",
    "latestVersion": "1.1.0",
    "files": [
      {
        "filename": "plugins/AddColorCode.auf",
        "isDirectory": false,
        "isOptional": false
      }
    ],
    "releases": [
      {
        "version": "1.1.0",
        "integrity": {}
      }
    ],
    "dependencies": [
      "aviutl1.10",
      "exedit0.92"
    ]
  }
]
```

### タイムスタンプ
2022/12/18 0:54:40

### LINKED ISSUE #441: 【パッケージの更新】Curve Editor
### 名前
みまらか

### データ
``` xml
[
  {
    "id": "mimaraka/curveeditor",
    "developer": "みまらか",
    "downloadURLs": [
      "https://github.com/mimaraka/aviutl-plugin-curve_editor/releases/latest"
    ],
    "pageURL": "https://github.com/mimaraka/aviutl-plugin-curve_editor",
    "name": "Curve Editor",
    "overview": "様々なイージングをウィンドウ上で編集",
    "description": "様々なイージングをウィンドウ上で編集できるAviUtlプラグイン&スクリプトです。",
    "latestVersion": "v1.0.4",
    "files": [
      {
        "filename": "plugins/curve_editor.auf",
        "archivePath": "curve_editor-v1.0.4"
      },
      {
        "filename": "script/@Curve Editor.tra",
        "archivePath": "curve_editor-v1.0.4",
        "isDirectory": false,
        "isOptional": false
      }
    ],
    "releases": [
      {
        "version": "v1.0.4",
        "integrity": {
          "archive": "sha384-pJnAaRoCTmkkcAnF3mL0kFGUq/l96WBb+GC2bicovFj9Y4CSJ3oaI7RkkNYtMnR0",
          "file": [
            {
              "hash": "sha384-fVBG3Hastb5jnmZ1hZwY2OwY32eRdLK16M9vhhhWRub4+MmLwkyVnvAZ2vvOJlpI",
              "target": "plugins/curve_editor.auf"
            },
            {
              "hash": "sha384-CI3i9TaatLv7c/Kt0FFOWnpggg4G9/VthIeZwQ+zkuAabreIGnPWBQ49vAxgA7S+",
              "target": "script/@Curve Editor.tra"
            }
          ]
        }
      }
    ]
  }
]
```

### タイムスタンプ
2022/12/26 18:14:11

### LINKED ISSUE #444: 【パッケージの更新】画像保存
## 名前
画像保存

## 概要
現在のフレーム、または選択アイテムを画像として保存
## 説明
現在のフレーム、または選択アイテムを画像として保存します。
## 開発者
hebiiro
## 紹介ページURL
https://github.com/hebiiro/AviUtl-Plugin-SaveImage
## ダウンロードページURL
https://github.com/hebiiro/AviUtl-Plugin-SaveImage/releases/tag/1.1.0
## ファイル名
SaveImage.auf

こんな感じでしょうか？
```
{
      "id": "hebiiro/AviUtl-Plugin-SaveImage",
      "name": "画像保存",
      "overview": "現在のフレーム、または選択アイテムを画像として保存",
      "description": "現在のフレーム、または選択アイテムを画像として保存します。",
      "developer": "hebiiro",
      "pageURL": "https://github.com/hebiiro/AviUtl-Plugin-SaveImage",
      "downloadURLs": [
        "https://github.com/hebiiro/AviUtl-Plugin-SaveImage/releases/tag/1.1.0"
      ],
      "latestVersion": "v1.1.0",
      "files": [
        {
          "filename": "plugins/SaveImage.auf",
          "archivePath": "SaveImage/"
        },
      ],
      "releases": [
        {
          "version": "v1.1.0",
          "integrity": {
            "archive": "sha384-c44e810a23ce19bdfa6194be52d244288b04b6f1dadc3099b28d2512cb84bedde2e53c08665f71d2ed2804023ca7cbba",
            "file": [
              {
                "target": "plugins/SaveImage.auf",
                "hash": "sha384-c44e810a23ce19bdfa6194be52d244288b04b6f1dadc3099b28d2512cb84bedde2e53c08665f71d2ed2804023ca7cbba"
              }
            ]
          }
        },
        {
          "version": "v1.0.0",
          "integrity": {
            "file": [
              {
                "target": "plugins/SaveImage.auf",
                "hash": "sha384-969b53f8ee10ccac3ff3a0120fc45bcef3c3c59850a4d0bd49320e0064e960fec309e18a42cadae4705be5c17dbe31f3"
              }
            ]
          }
        },
      ]
    },
    ```

### LINKED ISSUE #445: 【パッケージ更新】改変版Auls終了確認
## 名前
改変版Auls終了確認
## 概要
終了確認プラグイン
## 説明
[Auls http://auls.client.jp/]さんの終了確認プラグインを勝手に改変したものです。
## 開発者
[かろてら](https://scrapbox.io/karoterra/)
## 紹介ページURL
https://scrapbox.io/karoterra/%E6%94%B9%E5%A4%89%E7%89%88Auls%E7%B5%82%E4%BA%86%E7%A2%BA%E8%AA%8D
## ダウンロードページURL
https://scrapbox.io/karoterra/%E6%94%B9%E5%A4%89%E7%89%88Auls%E7%B5%82%E4%BA%86%E7%A2%BA%E8%AA%8D
## ファイル名
auls_confirmclose.auf

### LINKED ISSUE #471: 【パッケージの更新】角縁取り
### 名前
mitosagi

### データ
``` xml
[
  {
    "id": "binetsu/SharpBorder",
    "developer": "微熱",
    "downloadURLs": [
      "https://www.dropbox.com/sh/96vgf2p4vji4x9a/AAD4m0XHt9oJKna0aWEWRD79a?dl=0"
    ],
    "pageURL": "https://www.nicovideo.jp/watch/sm41265376",
    "name": "角縁取り",
    "overview": "角をしっかり尖らせて縁取りする",
    "description": "Adobe Illustratorなどの縁取りみたいに、角をしっかり尖らせて縁取りするスクリプトです。",
    "latestVersion": "2022/10/27",
    "files": [
      {
        "filename": "opencv_world460.dll"
      },
      {
        "filename": "script/binetsu/角縁取り.anm"
      },
      {
        "filename": "script/binetsu/SharpBorder.dll"
      }
    ],
    "releases": [
      {
        "version": "2022/10/27",
        "integrity": {
          "archive": "sha384-vrZOlrotjLwSFhc59GD4AibLUI3U8tuy/BqMk/POFEerHSflQdKxVCDBq0lYBTCW",
          "file": [
            {
              "hash": "sha384-XM9nIKIhOTW/Lfoi4jEfPNmYyifym4Z3XHkHdIXD6RKtntdpl/xY4fIZ7mZOCRI8",
              "target": "opencv_world460.dll"
            },
            {
              "hash": "sha384-BrArhuXInFPghUJCKAle0z/W5yrQcG0zpUQ9O2n22/s7XrO/e84NapNU6CZtE03E",
              "target": "script/binetsu/角縁取り.anm"
            },
            {
              "hash": "sha384-ez90ymnBFqqIwgVCQjJzMPt7rnmsooKMXiPcMAB8SK9q5Zzah8YdpH8qmtKOLY//",
              "target": "script/binetsu/SharpBorder.dll"
            }
          ]
        }
      }
    ]
  }
]
```

### タイムスタンプ
2023/01/19 23:49:46

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Add user-submitted packages. Thanks to everyone who contributed package data.

Adding the `isHidden` attribute to `sort.js`.

## Related Issue

- closes #425 
- closes #435 
- closes #436 
- closes #441 
- closes #444 
- closes #445 
- closes #471 

## How Has This Been Tested?

- Package can be installed
- Package works on AviUtl

## Checklist:

- [ ] ~~Updated the modification date in `mod.xml`.~~ To avoid conflicts

### CODE REVIEW

File: v3/packages.json [hunk 1]
```diff
@@ -2295,6 +2363,57 @@
...
+            "archive": "sha384-LuEF3p+KrkgPPq4nS2vDTiie3eBN+cxGdSBvtOi99yvDV6CThYDfV6/RN1I2tOfi",
+            "file": [
+              {
+                "target": "plugins/LevelMeter.auf",
+                "hash": "sha384-zXBAx2/+zUu/JtxQWulBQIF3N5SidztEsBYQbc0Z08gPQR2rhsjUZvcYYnenR0nM"
+              },
+              {
+                "target": "plugins/LevelMeter.ini",
+                "hash": "sha384-OLBgp1GsljhM2TJ+sbHjaiH9txEUvgdDTAzHv2P24donTt6/529l+9Ua0vFImLlb"
+              }
```
Comment (hal-shu-sato) [inline v3/packages.json:2395]: The `*.ini` file is a configuration file, so we should not include it in `integrity`.

File: v3/packages.json [hunk 2]
```diff
@@ -2295,6 +2363,57 @@
...
+            "archive": "sha384-9t4WxpSz8g5tlAmLH+J7jkXfXlu5533r76Xrar+hrdcN1pIuZ3iqbygn8XHgHkja",
+            "file": [
+              {
+                "target": "plugins/LevelMeter.auf",
+                "hash": "sha384-QnxVgpbXI5kArpAY6NgxYde4sSd764NNGjYXHjav08zD3gr9MrYZgGfL9KvTta8A"
+              },
+              {
+                "target": "plugins/LevelMeter.ini",
+                "hash": "sha384-OLBgp1GsljhM2TJ+sbHjaiH9txEUvgdDTAzHv2P24donTt6/529l+9Ua0vFImLlb"
+              }
```
Comment (hal-shu-sato) [inline v3/packages.json:2411]: Same as L2392-2395

File: v3/packages.json [hunk 3]
```diff
@@ -3734,6 +3881,47 @@
...
+      "description": "拡張編集に未保存のデータがある場合、プロジェクトを閉じる前に確認メッセージを表示します。Auls( http://auls.client.jp/ )さんの終了確認プラグインを勝手に改変したものです。",
+      "developer": "karoterra",
+      "originalDeveloper": "yu_noimage_",
+      "dependencies": ["exedit0.92"],
+      "pageURL": "https://scrapbox.io/karoterra/%E6%94%B9%E5%A4%89%E7%89%88Auls%E7%B5%82%E4%BA%86%E7%A2%BA%E8%AA%8D",
+      "downloadURLs": [
+        "https://scrapbox.io/karoterra/%E6%94%B9%E5%A4%89%E7%89%88Auls%E7%B5%82%E4%BA%86%E7%A2%BA%E8%AA%8D"
+      ],
+      "latestVersion": "v1.2",
+      "files": [{ "filename": "plugins/auls_confirmclose.auf" }],
```
Comment (hal-shu-sato) [inline v3/packages.json:3897]:
```suggestion
      "files": [{ "filename": "auls_confirmclose.auf" }],
```
The original version should be placed in the same directory as `aviutl.exe`, so we should do the same.

File: v3/packages.json [hunk 4]
```diff
@@ -2295,6 +2363,57 @@
...
+      "name": "レベルメーター",
+      "overview": "音声のレベルメーター",
+      "description": "Aviutl内に音声のレベルメーターを表示します。",
+      "developer": "hebiiro",
+      "dependencies": ["exedit0.92"],
+      "pageURL": "https://github.com/hebiiro/AviUtl-Plugin-LevelMeter#readme",
+      "downloadURLs": [
+        "https://github.com/hebiiro/AviUtl-Plugin-LevelMeter/releases/latest"
+      ],
+      "latestVersion": "1.4.0",
```
Comment (hal-shu-sato) [inline v3/packages.json:2377]: Sorry for the late review.
The new version is released and the `LevelMeter` directory has been added so could you add it?

**Review by hal-shu-sato — CHANGES_REQUESTED** [review]
Sorry for the late review.
I have some points to request changes.

**Comment by mitosagi** [comment]
Corrected.

**Comment by mitosagi** [comment]
お時間があるときに確認していただけるとうれしいです。

**Comment by mitosagi** [comment]
お時間があるときに確認していただけるとうれしいです。

**Review by hal-shu-sato — APPROVED** [review]
It works fine.
Sorry for the late review.

**Pull request merged**